content/blog/math-in-javascript/index.md

```diff
@@ -0,0 +1,61 @@
+---
+{
+    title: "Math in JavaScript",
+    description: "Math in JavaScript may seem simple and sometimes easy, but sometimes what seems right is wrong.",
+    published: '',
```

Comment:
This should be a date string 😅

Reply:
I don't know what to put there. I'm not good with time.

Reply:
Also do I have to go and edit these in myself or just click resolve right here?